.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
       - id: check-yaml
         args: [--allow-multiple-documents]
       - id: check-toml
-      - id: check-added-large-files
+      #- id: check-added-large-files
```

Comment:
why this comment?

Reply:
The pick.lock file unfortunately is too big, which triggers this linting rule

Reply:
We can skip the file e.g.

```python
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.6.0
    hooks:
      - id: debug-statements
      - id: trailing-whitespace
      - id: end-of-file-fixer
      - id: check-yaml
      - id: check-added-large-files
        exclude: ^docs/source/notebooks/
```

Reply:
Done, thanks!